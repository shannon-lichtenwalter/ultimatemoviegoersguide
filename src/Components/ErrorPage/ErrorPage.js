import React from 'react';

class ErrorPage extends React.Component{
  render(){
    return(
      <section>
        <h2> Oops! Something has gone wrong! Please return to the home page and try again...</h2>
      </section>
    )
  }
}

export default ErrorPage;