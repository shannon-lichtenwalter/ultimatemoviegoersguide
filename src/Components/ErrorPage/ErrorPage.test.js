import React from 'react';
import ReactDOM from 'react-dom';
import ErrorPage from './ErrorPage';
import { mount } from 'enzyme';
import toJson from 'enzyme-to-json';

describe('ErrorPage Component', () => {
  it('renders without crashing', () => {
    const div = document.createElement('div');
    ReactDOM.render
      (<ErrorPage />, div);
    ReactDOM.unmountComponentAtNode(div);
  }); 

  it('Displays the ErrorPage correctly when rendered', () => {
    const wrapper = mount(<ErrorPage />);
    expect(toJson(wrapper)).toMatchSnapshot()
  });
});
