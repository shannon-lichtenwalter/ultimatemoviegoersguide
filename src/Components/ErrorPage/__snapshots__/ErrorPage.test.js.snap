// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`ErrorPage Component Displays the ErrorPage correctly when rendered 1`] = `
<ErrorPage>
  <section>
    <h2>
       Oops! Something has gone wrong! Please return to the home page and try again...
    </h2>
  </section>
</ErrorPage>
`;
